src/main/resources/files.vm

```diff
@@ -1,54 +1,74 @@
 <html>
-  <head>
-      <title>$aname : Bag $bname</title>
-      <link rel="stylesheet" type="text/css" href="/style.css">
-      <script type="text/javascript" src="/main.js"></script>
-  </head>
-  <body onload="init();">
-    <div class="breadcrumbs"><a href="/accessions">Accessions</a> <span class="hierarchy-separator" /> <a href="/accessions/$aname">$aname</a> <span class="hierarchy-separator" /><h1>$bname</h1></div>
-    <dl>
-        <dt>Title</dt>
-        <dd>$btitle</dd>
-    </dl>
-    <p>Image set description stub.</p>
-    <p id="imageSetCount"></p>
-    <div style="float:left;">
-	    <table style="float:left;">
-	        <tr><th>File Path</th><th>Size</th><th>Mime Type</th><th>Add</th></tr>
-	        #foreach( $f in $files )
-	            #set( $furi = $f["file"] )
-	            #if ($f["size"] == "")
-	              <tr class="missing-file">
-	                <td>$f["path"]</td>
-	                <td>(not in fedora)</td>
-	            #else
-	              <tr>
-	                <td><a href="$furi">$f["path"]</a></td>
-	                <td>$f["size"]</td>
-	            #end
-	              <td>$f["mime"]</td>
-	              <td><button type="button" title="Add image to selection" onclick="addRow(&quot;$f["path"]&quot;,&quot;$f["file"]&quot;,&quot;$f["mime"]&quot;)">Add</button></td>
-	            </tr>
-	        #end
-	    </table>
-	    
-	    <table id="selectionTable">
-	    	<tr>
-	    		<th>Move</th>
-	    		<th>Selected Files</th>
-	    		<th>Remove</th>
-	    		<th style="display: none;">D</th>
-	    	</tr>
-	    	<tr>
-	    		<td colspan="4">
-	    			<button type="button" id="createButton" title="Create image set from selection" onClick="checkName()">Create Image Set</button>
-	    			<input type="text" placeholder="Image Set Name" title="Image set name" id="imageSetName" onKeyUp="disableCreateButton();" />
-	    		</td>
-	    	</tr>
-	    </table>
-	    
-	    <p id="p"></p>
-	    <p id="debug"></p>
-    </div>
-  </body>
+<head>
+	<title>$aname : Bag $bname</title>
+	<link rel="stylesheet" type="text/css" href="/style.css">
+	<script type="text/javascript" src="/main.js"></script>
+</head>
+<body onload="init();">
+	
+	<div class="breadcrumbs">
+		<a href="/accessions">Accessions</a><span class="hierarchy-separator"></span>
+		<a href="/accessions/$aname">$aname</a><span class="hierarchy-separator"></span>
+	</div>
+	<div class="title">
+		<h1>Image Set Creator</h1>
+	</div>
+	<div class="title">
+		<h3>Bag ID:</h3>
+		<h2>$bname</h2>
+		<br>
+		<h3>Title:</h3>
+		<h2>$btitle</h2>
+	</div>
+
+	<p class="description">An <span class="bold">image set</span> is a user-defined collection of images from a particular bag. Here you can create image sets by selecting the desired images, naming the set, and hitting "Create Image Set." Once created, image sets can be viewed elsewhere through an IIIF viewer. You may edit previously created image sets through the link below.</p>
```

Comment:
This description (and the style modifications in general) are great, but it occurs to me that there are two main functions to this page: viewing the content of the bag and making image sets.  I think we need to do a better job of separating out those two functions so that people just doing the former aren't confused by the buttons and description about the later.

I can think of a few ways of accomplishing this.  
1.  Have two pages, each one dedicated to the one function or the other.   For the page that creates image sets it could be tailored for image set creation with this helpful description and all that.  It needn't display files at all that aren't eligible for image set inclusion.  The listing page was essentially what was there when you started working on this.

2.  Updating the listing page into an "image set creation" mode when the user clicks some button to indicate he/she wishes to do so.  The last column, the text, etc could all appear when the use clicks to initiate that mode.